### DISCUSSION TITLE
Enhance - Automatic Image Upscaling + Enhancement

### DISCUSSION THREAD

**Discussion by bobsponge76** [mashb1t/Fooocus#52] [Q&A] [answered]
Hi everyone,

I might be doing something wrong, but I'm encountering poor results when I select the "Enhance - Automatic Image Upscaling + Enhancement" function. Usually, the final image turns out worse than the original. I'm using Juggernaut 8.0 without applying any style. Any advice or suggestions on what might be causing this issue?

Thanks in advance!

**Comment by mashb1t** [comment, marked as answer]
Please ensure you use performance "Quality" or "Speed", which might be the cause of the issue you're experiencing. You may also experiment with upscale before/after enhance, see first tab in enhance.
